include/usb/class/usb_audio.h

```diff
@@ -17,4 +17,573 @@
 #ifndef ZEPHYR_INCLUDE_USB_CLASS_AUDIO_H_
 #define ZEPHYR_INCLUDE_USB_CLASS_AUDIO_H_
 
+#include <usb/usb_common.h>
+#include <device.h>
+#include <net/buf.h>
+#include <sys/util.h>
+
+#if defined(CONFIG_USB_AUDIO_HEADPHONES)
```

Comment:
L:25 ... L150 seems to be internal configuration dependent stuff, would it not be better to place it in `subsys/usb/audio/audio_internal.h` ?